app/lib/services.rb

```diff
@@ -3,7 +3,7 @@
 
 module Services
   def self.worldwide_api
-    @worldwide_api ||= GdsApi::Worldwide.new(Plek.new.find("whitehall-admin"))
```

Comment:
The `require "gds_api/worldwide"` at the top of the file might need changing to `require "gds_api"` for this change to work. If not, the `require "gds_api/worldwide"` should be redundant now.